contract/integration-tests/src/main.ava.ts

```diff
@@ -0,0 +1,404 @@
+    initialBalance: NEAR.parse("30 N").toJSON(),
+  });
+
+  // Deploy the contract
+  await contract.deploy(process.argv[2]);
+
+  // Import CA contract
+  const caContract = await root.importContract({
+    mainnetContract: "connected-accounts.near",
+    blockId: 88884919,
+    withData: true,
+  });
+
+  // Save state for test runs, it is unique for each test
+  t.context.worker = worker;
+  t.context.accounts = { root, contract, alice, bob, caContract };
+});
+
+test.afterEach(async (t) => {
+  // Stop Sandbox server
+  await t.context.worker.tearDown().catch((error) => {
+    console.log("Failed to stop the Sandbox:", error);
+  });
+});
+
+// ====== Global Objects ======
+
+const ACCOUNT_1 = {
+  id: "username",
+  originId: "social_network",
+  itemId: "https://social_network.com/username/status/14920813",
+};
+const globalIdAcc_1 = ACCOUNT_1.id + "/" + ACCOUNT_1.originId;
+const globalIdTwitterUser_1 = "teremovskii/twitter";
+const twitterUserItemId_1 = "https://twitter.com/teremovskii/status/1336406078574256135";
+const twitterUserItemId_2 = "https://twitter.com/teremovskii/status/1336";
+const user_wallet = "nikter.near";
+
+// ======= TESTS =======
+
+test("integration test", async (t) => {
+  // == TEST 1 ==
+  console.log("== TEST 1 ==: initialize contract");
+  // ============
+
+  const { alice, contract } = t.context.accounts;
+  await alice.call(contract, "initialize", {
+    ownerAccountId: alice.accountId,
+    caContractAddress: "connected-accounts.near",
+    maxAmountPerItem: "10000000000000000000000000",
+    maxAmountPerTip: "1000000000000000000000000",
+    network: "mainnet",
+  });
+
+  const ownerAccountId = await contract.view("getOwnerAccount", {});
+  const cAContractAddress = await contract.view("getCAContractAddress", {});
+  const maxAmountPerItem = await contract.view("getMaxAmountPerItem", {});
+  const cmaxAmountPerTip = await contract.view("getMaxAmountPerTip", {});
+
+  t.is(ownerAccountId, alice.accountId);
+  t.is(cAContractAddress, "connected-accounts.near");
+  t.is(maxAmountPerItem, "10000000000000000000000000");
+  t.is(cmaxAmountPerTip, "1000000000000000000000000");
+
+  // ============ CA contract initialization =============
+
+  // const STAKE = "1000000000000000000000"; // 0.001 NEAR
+  // await alice.call(caContract, "initialize", {
+  //   ownerAccountId: alice.accountId,
+  //   oracleAccountId: alice.accountId,
+  //   minStakeAmount: STAKE,
+  // });
+
+  // const ownerAccountId_caContract = await caContract.view("getOwnerAccount", {});
+  // const oracleAccountId_caContract = await caContract.view("getOracleAccount", {});
+  // const minStakeAmount_caContract = await caContract.view("getMinStakeAmount", {});
+
+  // t.is(ownerAccountId_caContract, alice.accountId);
+  // t.is(oracleAccountId_caContract, alice.accountId);
+  // t.is(minStakeAmount_caContract, STAKE);
+
+  // == TEST 2 ==
+  console.log("== TEST 2 ==: send tips to twitter account without connected accounts");
```

Comment:
Why it's not a separate test/step?

Reply:
Because here is one integration test with cross-cutting scenario. Separated tests have different default environment. It's not convenient is this case.